src/popup/views/RecoverAccount/index.tsx

```diff
@@ -78,20 +92,21 @@ const RecoverAccount = () => {
       validationSchema={RecoverAccountSchema}
     >
       {({ handleChange, isSubmitting, isValid }) => (
-        <Form>
+        <FullHeightForm>
           <Onboarding
             header="Recover wallet from backup phrase"
             icon={icon}
-            goBack={() => history.push({ pathname: "/" })}
+            isMaxHeaderLength
```

Comment:
Removed this because the design comp doesn't include a back button

Reply:
sorry! same thing - let's leave the back button in: https://www.figma.com/file/uIoeq72R2XjiWjGyVGiPG6/Lyra-wireframes?node-id=0%3A1

Reply:
Added back